modules/andes-core/broker/src/main/java/org/wso2/andes/thrift/SlotManagementServiceImpl.java

```diff
@@ -87,6 +88,21 @@ public boolean deleteSlot(String queueName, SlotInfo slotInfo, String nodeId) th
         }
     }
 
+    /**
+     * Used to check the health of the connection taken from thrift connection pool.
+     *
+     * @param healthCheckStr The string received to healthCheck.
+     * @return received string is the expected or not.
+     * @throws TException thrift exception.
+     */
+    @Override
+    public boolean healthCheck(String healthCheckStr) throws TException {
+        if (healthCheckStr != null && healthCheckStr.equals(Constants.HEALTH_CHECK)) {
```

Comment:
Instead of using the if else clause, what if we return the condition directly? WDYT?

Reply:
implementation is now changed